binary-tarball-tests/ps/settings.py

```diff
@@ -119,61 +120,60 @@
   'bin/mysqldumpslow',
   'bin/mysql_config',
 ]
-ps81_plugins = (
+ps8x_plugins = (
   ('mysql_no_login','mysql_no_login.so'),('validate_password','validate_password.so'),
   ('version_tokens','version_token.so'),('rpl_semi_sync_master','semisync_master.so'),('rpl_semi_sync_slave','semisync_slave.so'),
   ('group_replication','group_replication.so'),('clone','mysql_clone.so'),
-  ('procfs', 'procfs.so'), ('authentication_ldap_sasl','authentication_ldap_sasl.so'),
-  ('authentication_fido','authentication_fido.so')
+  ('procfs', 'procfs.so'), ('authentication_ldap_sasl','authentication_ldap_sasl.so')
 )
-ps81_functions = (
+ps8x_functions = (
   ('version_tokens_set', 'version_token.so', 'STRING'),('version_tokens_show', 'version_token.so', 'STRING'),('version_tokens_edit', 'version_token.so', 'STRING'),
   ('version_tokens_delete', 'version_token.so', 'STRING'),('version_tokens_lock_shared', 'version_token.so', 'INT'),('version_tokens_lock_exclusive', 'version_token.so', 'INT'),
   ('version_tokens_unlock', 'version_token.so', 'INT'),('service_get_read_locks', 'locking_service.so', 'INT'),('service_get_write_locks', 'locking_service.so', 'INT'), ('service_release_locks', 'locking_service.so', 'INT')
 )
 
-ps81_components = (
+ps8x_components = (
   'component_masking_functions', 'component_binlog_utils_udf', 'component_percona_udf', 'component_audit_log_filter', 'component_keyring_vault'
 )
 
-ps81_files = (
+ps8x_files = (
   'lib/libcoredumper.a', 
   'lib/mysqlrouter/private/libmysqlrouter_http.so.1', 'lib/mysqlrouter/private/libmysqlrouter.so.1', 'lib/libmysqlservices.a',
-  'lib/libperconaserverclient.a', 'lib/libperconaserverclient.so.22.0.0' ,'lib/mysql/libjemalloc.so.1',
+  'lib/libperconaserverclient.a', 'lib/libperconaserverclient.so.22.1.0' ,'lib/mysql/libjemalloc.so.1',
   'lib/plugin/ha_rocksdb.so', 'lib/plugin/auth_pam.so', 'lib/plugin/auth_pam_compat.so',
   'lib/plugin/keyring_file.so','lib/plugin/component_binlog_utils_udf.so',
   'lib/plugin/keyring_udf.so', 'lib/plugin/component_keyring_vault.so', 'lib/plugin/component_binlog_utils_udf.so',
   'lib/plugin/component_audit_log_filter.so', 'lib/plugin/component_masking_functions.so'
 )
 
-ps81_symlinks = (
-  ('lib/libperconaserverclient.so.22','lib/libperconaserverclient.so.22.0.0'),
-  ('lib/libperconaserverclient.so','lib/libperconaserverclient.so.22.0.0'),('lib/mysql/libjemalloc.so','lib/mysql/libjemalloc.so.1')
+ps8x_symlinks = (
+  ('lib/libperconaserverclient.so.22','lib/libperconaserverclient.so.22.1.0'),
+  ('lib/libperconaserverclient.so','lib/libperconaserverclient.so.22.1.0'),('lib/mysql/libjemalloc.so','lib/mysql/libjemalloc.so.1')
 )
 
 #####
 
-if ps_version_major == '8.1':
-    ps_binaries = ps81_binaries
-    ps_executables = ps81_executables
-    ps_plugins = ps81_plugins
-    ps_functions = ps81_functions
-    ps_files = ps81_files
-    ps_symlinks = ps81_symlinks
-    ps_components = ps81_components
+if re.match(r'^8\.[1-3]$', ps_version_major):
```

Comment:
Why do we match only 1-3? We will have 8.4 etc in some time..

Reply:
ok let me add till 9

Reply:
fixed